tests/test_job.py

```diff
@@ -108,6 +109,7 @@ def test_submit_job_url_with_success(self, mock_session, make_mock_response):
             'speaker_channels_count': 1,
             'filter_profanity': True,
             'remove_disfluencies': True,
+            'language': LANG,
```

Comment:
it makes no difference in execution but I would order these to match the order of the parameters with language coming last. Make this change here and below and in the submit local file unit test as well.